.buildkite/scripts/serverless/create_deploy_tag/prepared_exec.ts

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
We might not need this module here, but it came in handy for me while testing locally